third_party/move/move-prover/src/lib.rs

```diff
@@ -280,9 +280,10 @@ pub fn create_and_process_bytecode(options: &Options, env: &GlobalEnv) -> Functi
     let output_dir = Path::new(&options.output_path)
         .parent()
         .expect("expect the parent directory of the output path to exist");
-    let output_prefix = options.move_sources.first().map_or("bytecode", |s| {
-        Path::new(s).file_name().unwrap().to_str().unwrap()
-    });
+    let output_prefix = options.move_sources.first().map_or_else(
```

Comment:
const literals are cheaper than functions?

Reply:
true, missed this one, usually the linter would complain in this case. will revert.

Reply:
you have this in multiple places as far as I can see

Reply:
will do a pass. For sure left a bunch unchanged and got linter errors and reverted on some. string literal def can stay in or(

Reply:
I think I got all of them